sig-security/self-assessment.md

```diff
@@ -0,0 +1,228 @@
+| ---------------------------- | ------------------------------------------------------------ |
+| Security file                | [SECURITY.md](https://github.com/kubeedge/kubeedge/blob/master/SECURITY.md) |
+| Default and optional configs | [cloudcore](https://github.com/kubeedge/kubeedge/blob/master/pkg/apis/componentconfig/cloudcore/v1alpha1/default.go)<br />[edgecore](https://github.com/kubeedge/kubeedge/blob/master/pkg/apis/componentconfig/edgecore/v1alpha1/default.go) |
+
+## Overview
+
+KubeEdge is an open source system for extending native containerized application orchestration capabilities to hosts at the edge. It's built upon Kubernetes and provides fundamental infrastructure support for networking, application deployment and metadata synchronization between cloud and edge.
+
+Since joining CNCF, KubeEdge has attracted more than [1000+ Contributors](https://kubeedge.devstats.cncf.io/d/18/overall-project-statistics-table?orgId=1) from 80+ different Organizations with 3,900+ Commits; got 5,600+ Stars on Github and 1,600+ Forks. It has been adopted by China Mobile, China Telecom, Raisecom, WoCloud, Xinghai IoT, KubeSphere, HUAWEI CLOUD, Harmony Cloud, DaoCloud, SAIC Motor, SF Express, etc.
+
+### Background
+
+As 5G and AI technologies grow, enterprises are increasingly demanding intelligent upgrade, and the application scenarios of edge computing are becoming more and more extensive. It is challenging to realize unified management and control of edge computing resources and cloud-edge synergy, such as how to run intelligent applications and algorithms on edge devices with limited resources(e.g. cameras and drones), how to solve the problems caused by the access of mass heterogeneous edge devices in intelligent transportation and intelligent energy, and how to ensure high reliability of services in off line scenarios.
+
+KubeEdge provides solutions for cloud-edge synergy and has been widely adopted in industries including transportation, energy, Internet, CDN, Manufacturing, Smart Campus, etc. KubeEdge provides:
+
+- Seamless Cloud-Edge Communication for both metadata and data.
+- Edge Autonomy: Autonomous operation of Edge even during disconnection from cloud.
+- Low Resource Readiness: KubeEdge can work in constrained resource situations (low memory, low bandwidth, low compute).
+- Simplified Device Communication: Easy communication between application and devices for IOT and IIOT.
+
+### Goal
+
+The main goals of KubeEdge are as follows:
+
+- Building an open edge computing platform with cloud native technologies.
+- Helping users extending their business architecture, applications, services, etc. from cloud to edge but ensuring same user experience.
+- Implementing extensible architecture based on Kubernetes.
+- Integrating with CNCF projects, including (but not limited to) containerd, cri-o, Prometheus, Envoy, etc.
+- Seamlessly develop, deploy, and run complex workloads at the edge with optimized resources.

```

Comment:
what do the “optimized resources” mean? a little confusing. as i understand it, it means that KubeEdge can schedule the workloads to the most optimized resources. but this can not explain "develop".

Reply:
Not only for deploying, but also we can develop applications with optimized resources with the help of KubeEdge.